pinot-core/src/main/java/com/linkedin/pinot/core/segment/index/loader/invertedindex/InvertedIndexHandler.java

```diff
@@ -91,7 +91,31 @@ public void createInvertedIndices()
 
   private void createInvertedIndexForColumn(ColumnMetadata columnMetadata)
       throws IOException {
+    // Validation check
     String column = columnMetadata.getColumnName();
+    int totalDocs = columnMetadata.getTotalDocs();
+    if (totalDocs > OffHeapBitmapInvertedIndexCreator.MAX_NUM_ENTRIES) {
+      LOGGER.warn(
```

Comment:
I agree it's hard to track this performance problem if inverted indexes are not created. Actually adding a counter (metric) here does not help that much, and it's not very possible to pass the ServerMeter all the way down to this method. Should we just throw the exception, and let CRC check catch this?